libs/@guardian/libs/src/measure/startMeasure.ts

```diff
@@ -0,0 +1,46 @@
+import type { TeamName } from '../logger/@types/logger';
+import type {} from './@types/measure';
+import { serialise } from './serialise';
+
+/** For browser which do not fully support the performance API */
+const fallback: ReturnType<typeof startMeasure> = {
+	endMeasure: () => -1,
+};
+
+/**
+ * Helper to measure the duration between two events.
+ *
+ * Once ended, measures are appended to the list of `PerformanceMeasure`
+ * and can be retrieved with `performance.getEntriesByType('measure')`
+ *
+ * @example
+ * const { endMeasure } = startMeasure('dotcom', 'fetch');
+ * await fetch('https://www.theguardian.com/uk.json');
+ * const duration = endMeasure();
+ */
+export const startMeasure = (
```

Comment:
Do these functions need to be exported from the main `index.ts`?

Reply:
Ah yeah I thought about it and then completely forgot!